.github/workflows/scripts/biome_disable_check.py

```
@@ -0,0 +1,160 @@
+                        file_path = os.path.join(root, file_name)
+                        if has_biome_ignore(file_path):
+                            print(
+                                f"""File {file_path} contains biome-ignore
+                                statement.""",
+                            )
+                            biome_found = True
+
+    return biome_found
+
+
+def arg_parser_resolver():
```

Ruff (0.11.9)

Missing return type annotation for public function `arg_parser_resolver`

(ANN201)